package/MDAnalysis/core/topologyattrs.py

```diff
@@ -202,6 +202,17 @@ def stub_method(self, *args, **kwargs):
         )
         raise NoDataError(message)
 
+    def stub_method(self, *args, **kwargs):
```

Comment:
I think this has been duplicated from above.

Reply:
Yes sorry! I had to remove it at one point to get my tests to run then added it back.